libs/tradesman/lib/pages/advert_details.dart

```diff
@@ -42,127 +43,101 @@ class TradesmanJobDetails extends StatelessWidget {
-                                                  BorderRadius.circular(7),
-                                              borderSide: const BorderSide(
-                                                color: Colors.black,
-                                                width: 1.0,
-                                              ),
-                                            ),
-                                            focusedBorder: OutlineInputBorder(
-                                              borderRadius:
-                                                  BorderRadius.circular(7),
-                                              borderSide: const BorderSide(
-                                                color: Colors.orange,
-                                                width: 2.0,
-                                              ),
-                                            ),
-                                          ),
-                                        ),
-                                      ),
-                                      Padding(
-                                        padding: const EdgeInsets.all(15.0),
-                                        child: Row(
-                                          mainAxisAlignment:
-                                              MainAxisAlignment.center,
-                                          children: [
-                                            ButtonWidget(
-                                                text: "Submit",
-                                                function: () {}),
-                                            const Padding(
-                                                padding: EdgeInsets.all(3)),
-                                            ButtonWidget(
-                                                text: "Cancel",
-                                                color: "light",
-                                                border: "lightBlue",
-                                                function: () {})
-                                          ],
-                                        ),
-                                      )
-                                    ],
-                                  );
-                                });
-                          })),
+                
 
-                  //view bids
-                  Positioned(
-                      top: 95,
-                      child: ButtonWidget(
-                          text: "View Bids",
-                          color: "light",
-                          function: vm.pushViewBidsPage)),
+                //*************BOTTOM BUTTONS**************//
+                AuthButtonWidget(
+                    text: "Place Bid",
+                    function: () {
+                      DarkDialogHelper.display(context,
+                      PlaceBidPopupWidget(store: store), 1000.0);
+                      // showModalBottomSheet(
```

Comment:
Do we need this commented out code for later because its a lot.

Reply:
Oh I made a mistake, the current popup should be replaced with the commented out code, will quickly fix that